Source/astcenc_compress_symbolic.cpp

```diff
@@ -1380,7 +1385,8 @@ float compress_symbolic_block(
 				}
 			}
 
-			best_errorvals_in_modes[4 * (partition_count - 2) + 5 + i] = best_errorval_in_mode;
+			// Modes 5, 6, 8, 9, 11, 12
+			best_errorvals_in_modes[3 * (partition_count - 2) + 5 + i] = best_errorval_in_mode;
```

Comment:
Would it make sense to bounds check this calculation?

Reply:
For now I will leave this as the "minimal" edit to the existing code. I want to refactor this shortly; few of the values are actually used.